dev/test/document.ts

```diff
@@ -1150,6 +1150,7 @@ describe('set document', () => {
           request,
           set({
             document: document('documentId'),
+            mask: updateMask(),
```

Comment:
turns out we had the test all this time, but I deleted it from the unit tests back when I first added the proto :(